bitnami/spring-cloud-dataflow/Chart.yaml

```
@@ -53,4 +53,4 @@ maintainers:
 name: spring-cloud-dataflow
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/spring-cloud-dataflow
-version: 26.5.3
+version: 26.5.3+0.0.1
```

Please note chart version should be bumped following semver
```suggestion
version: 26.5.4
```

---

Done! Thanks!